.editorconfig

```diff
@@ -0,0 +1,19 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
originally added by `ember addon` but moved to the root of the project, so that it can act as the root config across all packages in the repo